cilium-cli/utils/features/features.go

```diff
@@ -399,8 +400,8 @@ func (fs Set) ExtractFromConfigMap(cm *v1.ConfigMap) {
 		Enabled: cm.Data[string(Multicast)] == "true",
 	}
 
-	fs[EncryptionStrictMode] = Status{
-		Enabled: cm.Data[string(EncryptionStrictMode)] == "true",
+	fs[EncryptionStrictEgress] = Status{
+		Enabled: cm.Data[string(EncryptionStrictMode)] == "true" || cm.Data[string(EncryptionStrictEgress)] == "true",
```

Comment:
Consider adding a comment here saying that the or-clause can be removed once 1.17 is EOL (since this cannot be removed when we remove the flag itself in the v1.19 dev cycle)